### ISSUE TITLE
add binary support for mysql

### ISSUE THREAD

**Issue by lal12** [Kononnable/typeorm-model-generator#31]
Hello,

I used this tool on a mysql database with a table containing a binary column. But it complains `Unknown column type:binary`, as far as I could quickly see in your code, there is binary support but just for mssql. So I guess it should be easy to fix?

Thanks
lal12

**Comment by Kononnable** [comment]
I don't see binary and point type supported for mysql driver. [Column types for mysql / mariadb](http://typeorm.io/#/entities/column-types-for-mysql--mariadb) 
Types should be supported by typeorm library first, than it can be implemented to model generator. Otherwise we will generate code that doesn't work.

There is an issue for adding point type to mysql driver typeorm/typeorm#517, but I'm not sure about binary. You should check typeorm issues and and add a new one if necessary.

**Comment by lal12** [comment]
I agree with you. I was mistaken and thought typeorm would support those types.
